Instagram Reels Bot/Helpers/IGComponentBuilder.cs

```diff
@@ -13,28 +13,28 @@ internal class IGComponentBuilder
         /// The Response from the Instagram processor.
         /// </summary>
         private InstagramProcessorResponse Response;
-        private string Requester;
+        private ulong RequesterId = 0;
```

Comment:
Why not just leave uninitialized and check for null?

Reply:
if there is user with no user.id interacting with the command. it would set to 0 or `RequesterIsKnown` = `false`. can't test it though